vendor/parser-extended/parser.mly

```diff
@@ -724,6 +724,23 @@ let convert_layout_to_legacy_attr =
   | {txt = Immediate64; loc} -> mk ~loc "immediate64"
   | _ -> []
 
+(* NOTE: An alternate approach for performing the erasure of %call_pos and %src_pos
+   could have been doing it as a ppx transformation instead of performing the erasing
+   inside of ocamlformat. *)
+let transl_label ~pattern ~arg_label ~loc =
+  match arg_label, pattern.ppat_desc with
+  | ( Labelled l
+    , Ppat_constraint
+        (pat, { ptyp_desc = Ptyp_extension ({ txt = "call_pos"; loc = _ }, _); _ }) )
+    when Erase_jane_syntax.should_erase () ->
```

Comment:
Though I think this works as written, I'm a tiny bit worried about performance. I think it would be slightly better just to use `if Erase_jane_syntax.should_erase ()` somewhere before looking deeply at the syntax. (here and elsewhere in this file -- the similar code in the normalizer is fine, because that's already doing pattern matching against syntax)

Reply:
This sounds reasonable. Changed!